examples/storage/getobject.go

```diff
@@ -68,19 +70,54 @@ func main() {
 		log.Fatalf("NewClient: %s", err)
 	}
 
-	obj, err := client.Objects().Get(context.Background(), &storage.GetObjectInput{
-		ObjectPath: "/stor/books/dracula.txt",
+	// You must have this in Manta in order for this file to work.
+	// path := "/stor/books/dracula.txt"
+	path := "/stor/books/dracula.txt"
+
+	ctx := context.Background()
+	info, err := client.Objects().GetInfo(ctx, &storage.GetInfoInput{
+		ObjectPath: path,
+	})
+	if err != nil {
+		fmt.Printf("Could not find '%s'\n", path)
+		return
+	}
+
+	fmt.Println("--- HEAD ---")
+	fmt.Printf("Content-Length: %d\n", info.ContentLength)
+	fmt.Printf("Content-MD5: %s\n", info.ContentMD5)
+	fmt.Printf("Content-Type: %s\n", info.ContentType)
+	fmt.Printf("ETag: %s\n", info.ETag)
+	fmt.Printf("Date-Modified: %s\n", info.LastModified.String())
+
+	ctx = context.Background()
+	isDir, err := client.Objects().IsDir(ctx, path)
+	if err != nil {
+		log.Fatalf("failed to detect directory '%s'\n", err)
+		return
+	}
+
+	if isDir {
+		fmt.Printf("'%s' is a directory\n", path)
+	} else {
+		fmt.Printf("'%s' is a file\n", path)
```

Comment:
Can we convert both of these `%s` format verbs to `%q`?  Basically: `"'%s'` -> `"%q`  Faux quoting is a pet peeve, especially when we have `%q` floating around. :)

Reply:
👍 and you may have called me out on that before.